cmd/notary/tuf.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/docker/distribution/registry/client/auth/challenge"
 	"github.com/docker/distribution/registry/client/transport"
 	"github.com/docker/go-connections/tlsconfig"
+	"github.com/docker/go/canonical/json"
```

Comment:
let's import this aliased to `canonicaljson` for clarity. It reduces confusion when/if somebody is having type errors related to the `encoding/json` vs `.../canonical/json` libraries